tasks/gemini.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Gemini composer install
+  composer:
+    command: install
+    working_dir: "{{ crayfish_install_dir }}/Gemini"
+
+- name: Gemini migrate db
+  shell: php bin/console migrations:migrate
+  args:
+    chdir: "{{ crayfish_install_dir }}/Gemini"
+  register: gemini_migrate
+  changed_when: '"No migrations to execute." not in gemini_migrate.stdout'
```

Comment:
Without testing this seems wrong. Single quotes won't evaluate the `not in` inside the string
So could be maybe  

```YAML
changed_when: "gemini_migrate.stdou is not search('No migrations to execute')" 
````

? or any other option, but single quotes around everything seem to me that would evaluate to a string instead.

Reply:
It does indeed work as written. At least, it worked for my own tests and the Travis builds.

Another example of single-quotes encapsulating a condition for changed_when can be found in the "Convergence and command/shell" section (changed_when subsection) of the blog post "[Using Ansible's command and shell modules properly](http://willthames.github.io/2016/09/21/using-command-and-shell-in-ansible.html)".

Reply:
Are your sure it is not evaluating to false always? I believe you and makes sense for running shell commands, but for internal evaluations like {{ }} or not in, seems ambiguous. Will add a debug there and test.

Reply:
I just scoured the ansible documentation and found nothing that suggested single-quotes will always evaluate to a string.

It does look like I could have omitted the single-quote altogether; but I hesitate to change it on speculation when Travis is already content.

Reply:
Ok, i trust you, will approve. If ansible handles double quotes and single quotes as the same, by expanding statement and even variable evaluation, then i feel something is wrong with Ansible. This can lead to very problematic issues. And i may not be the only one https://zncb.me/blog/2016/03/11/sneaky-ansible-pitfalls/